python/mkdocs/docs/feature-support.md

```diff
@@ -42,6 +42,22 @@ The goal is that the python library will provide a functional, performant subset
 | Drop Namespace           |  X   |   X    |
 | Set Namespace Properties |  X   |   X    |
 
+## Catalog capabilities
+
+The following table projects the capabilities of each of the available Catalog implementations:
+
+|                        | Rest | Hive | Glue | DynamoDB |
+| ---------------------- | ---- | ---- | ---- | -------- |
+| CRUD Namespaces        | ✓    | ✓    | ✓    | ✓        |
+| CRUD Tables            | ✓    | ✓    | ✓    | ✓        |
+| Update format version  | ✓    | ✕    | ✕    | ✕        |
+| Alter table properties | ✓    | ✕    | ✕    | ✕        |
+| Alter schema           | ✓    | ✕    | ✕    | ✕        |
+| Alter partition spec   | ✓    | ✕    | ✕    | ✕        |
+| Alter sort order       | ✓    | ✕    | ✕    | ✕        |
```

Comment:
I think we should start with table properties and move on to the other updates after that.

Reply:
Done